medusa/model/recommended.py

```diff
@@ -0,0 +1,91 @@
+
+import datetime
+from sqlalchemy import create_engine, ForeignKey
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy import Boolean, Column,  DateTime, Integer, String
+from sqlalchemy.orm import relationship, sessionmaker
+from sqlalchemy.sql import func
+
+from medusa.indexers.indexer_config import (EXTERNAL_ANIDB, EXTERNAL_IMDB, EXTERNAL_TRAKT, INDEXER_TMDB,
+                                            INDEXER_TVDBV2, INDEXER_TVMAZE)
+
+engine = create_engine('sqlite:///:memory:', echo=True)
+
+Base = declarative_base()
+
+
+class Indexer(Base):
+    __tablename__ = 'indexer'
+
+    id = Column(Integer, primary_key=True, autoincrement=True)
+    name = Column(String(255))
+    mapped_to = Column(String(255))
+    identifier = Column(String(255))
+    indexer_id = Column(Integer)
+    is_indexer = Column(Boolean)
+
+
+class RecommededShow(Base):
+    __tablename__ = 'recommended'
```

Comment:
`recommended` is a poor choice of terms... it should be `recommendations` or `recommended_series`.  Personally my preference is `recommendations` as then its not just specifically series, it could also be movies in the future.